mig-frankfurt/dataelementhub.dal
# #11 clean up database

**What's in the PR**
* new table "validation"
* columns that stored validation information is removed from the element table
* the hidden column is moved from element to scoped_identifier
* scoped_identifier table no longer differentiates between the different value domain types (element still does)
* remove entries from element_relation table when scoped identifier is deleted (via trigger)
* unused tables (config and permissible_code) are removed

**How to test manually**
* ideally you have a well filled dehub database. then clone it into a new one (use as template..)
* run the scripts manually in the correct order on your cloned db
* check if the validation is correctly migrated to the new table
* also check if the new entries really make sense(**!**)
* this will massively affect dehub-model, so this will most likely become a new major version. This will **NOT** work if you use the currently released version of dehub-model/rest
